condensedinlinepanel/edit_handlers.py

```diff
@@ -176,7 +177,8 @@ def bind_to_model(self, model):
             'relation_name': self.relation_name,
             'related': related,
             'panels': self.panels,
-            'heading': self.label,
```

Comment:
I think we should leave the heading field called ``label`` so it's the same as Wagtail's builtin ``InlinePanel``.